docs/tests/timings/profile-method.js

```diff
@@ -0,0 +1 @@
+../../../test/timings/profile-method.js
```

Comment:
I may clean this up a bit- we've set Github pages to serve assets from the `docs` folder of the master branch, and web servers don't like to serve things from arbitrary locations on the hard drive.

As a general rule going forward, I'll try to encourage that the `docs` folder be a destination for built assets- something that we can wipe completely if need be and regenerate. (this is a crude but effective means of pruning dead assets, which can creep in whenever a build or template system is involved). 

Accordingly, I may move the pandoc files and scripts to another folder as part of post-merge cleanup.

Reply:
Makes good sense to me.